### ISSUE TITLE
Lakea Trigger Woodland Event

### ISSUE THREAD

**Issue by JohnWolfson** [JohnWolfson/Lakea-Stream-Assistant#8]
Lakea should be able to send data to the woodland to trigger events

Acceptance Criteria:

- Lakea should be able to send messages to the woodland
- The woodland should be able to log the received message to the console

**Comment by JohnWolfson** [comment]
This ticket is currently blocked, waiting on the results of #7 before implementing. It should be noted that the woodland is currently a prototype and a new project for this will be created at some point.

**Comment by IstalriSkolir** [comment]
Lakea has a websocket that the new Woodland project will be able to connect to when its started. The Stream Assistant still needs subscription service endpoints creating, however these will be reflected in new tickets. Archiving this ticket as its been superseded by these events.
